nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=import-self, invalid-name, unused-argument
+# pylint: disable=import-self, invalid-name, unused-argument, too-many-lines
```

Comment:
too-many-lines ??
Suggest to have logical split instead. If any block can't be reduced use local enable/disable instead.